opencti-platform/opencti-graphql/src/resolvers/attackPattern.js

```diff
@@ -18,19 +19,25 @@ import {
 } from '../domain/stixDomainObject';
 import { loadThroughDenormalized } from './stix';
 import { INPUT_KILLCHAIN } from '../schema/general';
+import { batchLoader } from '../database/middleware';
+
+const batchLoadSubAttackPatterns = batchLoader(batchChildAttackPatternsPaginated);
+const batchLoadParentAttackPatterns = batchLoader(batchParentAttackPatternsPaginated);
+const batchLoadIsSubAttackPattern = batchLoader(batchIsSubAttackPattern);
 
 const attackPatternResolvers = {
   Query: {
     attackPattern: (_, { id }, context) => findById(context, context.user, id),
     attackPatterns: (_, args, context) => findAll(context, context.user, args),
+    attackPatternsMatrix: (_, __, context) => getAttackPatternsMatrix(context, context.user),
```

Comment:
Could you please cover this new API with a resolver test ? My first idea would be to add a call in https://github.com/OpenCTI-Platform/opencti/blob/master/opencti-platform/opencti-graphql/tests/02-integration/02-resolvers/attackPattern-test.js